conf/defaults.ini

```diff
@@ -593,6 +593,8 @@ notification_timeout_seconds = 30
 # Default setting for max attempts to sending alert notifications. Default value is 3
 max_attempts = 3
 
+# Makes it possible to enforce a minimal interval between evaluations, to reduce load on the backend
+min_interval_seconds = 1
```

Comment:
Should be added to [documentation](https://grafana.com/docs/grafana/latest/installation/configuration/#alerting)